tests/integration/test_script_runner.py

```diff
@@ -304,6 +304,12 @@
         data_context_dir=None,
         data_dir="tests/test_sets/taxi_yellow_tripdata_samples/first_3_files",
     ),
+    IntegrationTestFixture(
+        name="how_to_create_and_edit_an_expectation_with_domain_knowledge",
+        user_flow_script="tests/integration/docusaurus/expectations/how_to_create_and_edit_an_expectationsuite_domain_knowledge.py",
+        data_context_dir=None,
+        data_dir="tests/test_sets/taxi_yellow_tripdata_samples/first_3_files",
```

Comment:
Maybe we can omit the data copying step since we don't need a batch of data (would speed up the test a little).

Reply:
good call.